guides/common/modules/con_configuring-active-directory-as-an-external-identity-provider-for-project.adoc

```diff
@@ -1,23 +1,22 @@
 [id="configuring-active-directory-as-an-external-identity-provider-for-project_{context}"]
 = Configuring Active Directory as an external identity provider for {Project}
 
-This section shows how to use direct Active Directory (AD) as an external authentication source for {ProjectServer}.
+If the base system of your {ProjectServer} is connected directly to Active Directory (AD), you can configure AD as an external authentication source for {Project}.
+Direct AD integration means that a Linux system is joined directly to the AD domain where the identity is stored. The following login methods are available for AD users:
```

Comment:
```suggestion
Direct AD integration means that a Linux system is joined directly to the AD domain where the identity is stored.
The following login methods are available for AD users:
```

Linux system = host aka. registered to Foreman? Foreman itself? any host?

Reply:
This is a general definition of Linux system. In this case, it's the base system of Foreman server. But I thought that since I was defining the term in general terms, I should use "Linux system" as a general term as well.